test/get-supported-languages.coffee

```diff
@@ -0,0 +1,17 @@
+humanizeDuration = require ".."
+getSupportedLanguages = humanizeDuration.getSupportedLanguages
+
+assert = require "assert"
+fs = require "fs"
+path = require "path"
+
+describe "getLanguageSupport", ->
+
+  definitionsPath = path.resolve(__dirname, "definitions")
+
+  languages = []
+  for file in fs.readdirSync(definitionsPath)
+    if path.extname(file) is ".csv"
+      languages.push path.basename(file, ".csv")
+
+  assert.deepEqual(languages, getSupportedLanguages())
```

Comment:
This might happen to work, but if the file system reads things in a different order, this test will needlessly fail. Let's change it to this:

```coffeescript
assert.deepEqual(languages.sort(), getSupportedLanguages().sort())
```